### ISSUE TITLE
PPSSPP won't start on versions past 1.5.4 after changing from AMD to NVIDIA, Vulkan check fails badly

### ISSUE THREAD

**Issue by Sandvichero** [hrydgard/ppsspp#11679]
I recently switched from an AMD GPU to a Nvidia one (gtx 2080) and now PPSSPP won't boot whatsoever and no error messages appear. This occurs any of the stable versions past 1.5.4 which boots and runs fine.
CPU: 17-3770K

**Comment by hrydgard** [comment]
Latest drivers? That's very weird.

Here's the full list of builds:
https://buildbot.orphis.net/ppsspp/index.php?m=fulllist

Any chance you could try a few between 1.5.4 and 1.6, and see if you can figure out which is the first one that doesn't start?

Thanks!

**Comment by Sandvichero** [comment]
Yes, I am on the latest drivers.

**Comment by hrydgard** [comment]
What happens if you go into memstick/PSP/SYSTEM/ppsspp.ini and change GPUBackend to 0? 

There might be some kind of driver conflict that happens with Vulkan when you've had AMD drivers installed and then install NVIDIA drivers. PPSSPP started to use Vulkan by default in 1.6..

Not quite sure how to fix this if that's the case...

**Comment by Sandvichero** [comment]
It can't create the ini because it can't start and if I take an ini from 1.5.4 with the GPUBackend set to zero it still doesn't boot.

**Comment by hrydgard** [comment]
Oh. And 1 or 2? I don't have any other good ideas, other than you trying a bunch of builds between 1.5.4 and 1.6 so we can find when the problem started.

**Comment by Sandvichero** [comment]
Setting it to 1 and 2 doesn't work either

**Comment by Sandvichero** [comment]
ppsspp-v1.5.4-692-g940d8b458-windows-amd64 The First one that won't boot at all

ppsspp-v1.5.4-86--to--689 Shows up on a black screen and closes

ppsspp-v1.5.4-83-gcaf9eca2d-windows-amd64 Runs properly

**Comment by Sandvichero** [comment]
GPU: Default to Vulkan if possible

On the first build that stops working

**Comment by hrydgard** [comment]
Thanks. This means that somehow our checks for Vulkan availablility fails so badly on your system, and my attempt to "improve vulkan detection" in 692 just made it worse on your machine.

Very strange, but at least somewhere to start looking - or looking for workarounds. Thanks for your testing.

**Comment by hrydgard** [comment]
Is there anything left of your AMD drivers to uninstall?

**Comment by Sandvichero** [comment]
I will run the AMD cleanup utility and retry.

**Comment by hrydgard** [comment]
If you haven't yet, hold off. Might want to try some fixes on your machine first, tomorrow.. at least we would like to detect this problem so we can warn other users.

**Comment by Sandvichero** [comment]
Oops I already ran it and PPSSPP works fine now, Thanks

**Comment by Sandvichero** [comment]
The normal uninstaller for Radeon doesn't delete all the drivers

**Comment by hrydgard** [comment]
Alright, well thanks for confirming.

**Comment by unknownbrackets** [comment]
So, the most safe thing we could do is never even try Vulkan on Windows until the user selects it:

```
Changing GPU backends requires PPSSPP to restart. Restart now?

Warning: some buggy GPU drivers may cause PPSSPP to crash.  If this happens, restart it manually.

[Yes] [No]
```

This would mean reducing the "may be available" check back to the pre-940d8b458 form on Windows, and keeping the Windows default as non-Vulkan.

-[Unknown]
